client/state/happychat/user/actions.js

```diff
@@ -21,10 +20,8 @@ export const setPresalePrecancellationAvailability = ( availability ) => ( {
 } );
 
 export const requestHappychatEligibility = () => ( dispatch ) => {
-	const clientSlug = config( 'client_slug' );
-	wpcom
-		.undocumented()
-		.getOlarkConfiguration( clientSlug )
+	wpcom.req
+		.get( '/help/olark/mine' )
```

Comment:
The `clientSlug` was never really sent, because `wpcom.req.get` ignores a `body` parameter. And the endpoint doesn't use it, as verified by checking the PHP implementation. So I'm removing it without replacement.